js/components/slider.js

```diff
@@ -0,0 +1,61 @@
+var React = require('react'),
+ReactDOM = require('react-dom'),
+Utils = require('./utils');
+
+var Slider = React.createClass({
+
+  componentDidMount: function() {
+    this.handleSliderColoring();
+  },
+
+  componentWillReceiveProps: function(nextProps) {
+    if (nextProps.value != this.props.value) {
+      this.handleSliderColoring();
+    }
+  },
+
+  handleSliderColoring: function(){
+    if (!Utils.isEdge()){
+      var inputs = document.querySelectorAll('input[type=range]');
```

Comment:
I think in that case changing the slider in the CC menu would then trigger the slider in the control bar to redraw as well, even though it hadn't changed.  Reassigning the oninput handler every time is a little strange as well, why not just perform the logic in oninput instead of assigning it and then triggering it?